BitFaster.Caching.UnitTests/Lru/ConcurrentLruTests.cs

```diff
@@ -359,6 +359,29 @@ public void WhenValueExpiresItIsDisposed()
             disposableValueFactory.Items[1].IsDisposed.Should().BeFalse();
         }
 
+        [Fact]
+        public void WhenValueExpiresItemRemovedEventIsFired()
+        {
+            var lruEvents = new ConcurrentLru<int, int>(1, 6, EqualityComparer<int>.Default);
+            lruEvents.ItemRemoved += OnLruItemRemoved;
+
+            for (int i = 0; i < 5; i++)
+            {
+                lruEvents.GetOrAdd(i, i => i);
+            }
+
+            removedItems.Count.Should().Be(1);
+            removedItems[0].Key.Should().Be(0);
```

Comment:
this is kind of a crappy test because default value of int is zero - change to reflect some values that would detect a failure. Also need equivalent test on TLru.